src/Actions.js

```diff
@@ -0,0 +1,118 @@
+// @flow
```

Comment:
don't use a default export for all actions. just individual exports for each function